add-chain/testdata/superchain/extra/addresses/sepolia/expected_plasma.json

```diff
@@ -0,0 +1,17 @@
+{
+  "AddressManager": "0xbbcfc68f590ae60feb1551f4c639e3eab5ea7ba9",
+  "Challenger": "0xe8D525b30dD4C406570d47396b5b237DB30F13F1",
+  "DAChallengeAddress": "0x3333333333333333333300000000000000000000",
```

Comment:
@trianglesphere the `DAChallengeAddress` is the only difference between this .json file produced by a vanilla chain versus a `plasma` chain. Is that correct?

Reply:
This would be the only address difference.